_posts/2019-07-15-hhvm-4.14.0.markdown

```diff
@@ -16,6 +16,9 @@ HHVM 4.14 is released! As 4.8 has long term support, it remains supported, as do
   `hhast-migrate --instanceof-is` can be used to automatically convert
   expressions of the form `$x instanceof SomeType` to expressions of the form
   `$x is SomeType`, `$x is SomeType<_>`, `$x is SomeType<_, _>` etc.
+  If the RHS of an `instanceof` is not a classname in sourcecode, but rather
+  an expression, HHAST will migrate to a `\is_a()` call. This does not
```

Comment:
Thanks; can you change this to say that it requires HHAST 4.14.4? 4.14.3 and below do not have this behavior.